src/sentry/ingest/transaction_clusterer/__init__.py

```diff
@@ -24,15 +24,15 @@ class NamespaceOption:
 class ClustererNamespace(Enum):
     TRANSACTIONS = NamespaceOption(
         name="transactions",
-        data="txnames",
+        data="txnames2",
```

Comment:
Introducing a new prefix here ensures that all collections of samples are created fresh, which is a prerequisite for adding the project ID to the new `...:projects` set.

If we didn't change the name here, we might risk never running the clusterer for existing redis sets again.